src/App.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable*/
```

Comment:
<img width="477" alt="image" src="https://github.com/FullStackkDev/Bookwriter-frontend/assets/36064269/ed195456-d0c8-4d0f-92c8-04e1e91f9819">
remove logout in mobile responsive